components/atom/label/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/prop-types */
```

Comment:
We need to update the lint config for TS files in order to remove this rule, not needed anymore.